### PR TITLE
checked sources

### PR DESCRIPTION
**Pull request by mira-gruenwald**
added sources and P0263 as author of one source text.
see #205

> Pull-requests should be ready to merge, and quick to review. Please help your reviewer by using this form. If you require a specific reviewer you can select them on the right. 

### Checklist
-   [x] The title line explains what these changes do
-   [x] All my changes are marked in the `last_modfied` column
-   [x] All formatting changes are intentional (dates, coordinates)
-   [x] My changes do not add empty rows
-   [x] I checked the modified table on GitHub (click `view`)

### CODE REVIEW

File: data/Person.csv [hunk 1]
```diff
@@ -292,20 +292,22 @@ P0163,Li,Shu,en,male,NULL,NULL,NULL,NULL,1883,1954,,,PL0081,,,https://zh.wikiped
 P0163,黎,澍,zh,male,NULL,NULL,NULL,NULL,1883,1954,,,PL0081,,,https://zh.wikipedia.org/wiki/%E9%BB%8E%E6%BE%8D,,https://www.renminbao.com/rmb/articles/2016/3/28/63207b.html,,,,2017-10-03,MG,2018-02-07,WH
 P0164,Chen,Jingrong,en,female,NULL,NULL,NULL,NULL,1917,1989,,,PL0006,,,https://zh.wikipedia.org/wiki/%E9%99%88%E6%95%AC%E5%AE%B9,,https://baike.baidu.com/item/%E9%99%88%E6%95%AC%E5%AE%B9,,,,2017-10-03,MG,2018-02-07,WH
 P0164,陈,敬容,zh,female,NULL,NULL,NULL,NULL,1917,1989,陈懿范,zh,PL0006,,,https://zh.wikipedia.org/wiki/%E9%99%88%E6%95%AC%E5%AE%B9,,https://baike.baidu.com/item/%E9%99%88%E6%95%AC%E5%AE%B9,,,,2017-10-03,MG,2018-02-07,WH
-P0166,Wu,,en,female,,,,,,,,,,,,,,,,,(Wu Hong's mother),2017-10-03,MG,2017-10-13,MG
-P0166,鸿,,zh,female,,,,,,,,,,,,,,,,,,2017-10-03,MG,2017-10-13,MG
+P0166,Wu,,en,female,,,,,,,,,,,,SS00029,116,,,,(Wu Hong's (P0029) mother),2017-10-03,MG,2018-02-19,MG
+P0166,巫,,zh,female,,,,,,,,,,,,SS00029,116,,,,,2017-10-03,MG,2018-02-19,MG
```
Comment (duncdrum) [inline data/Person.csv]: the 字 makes more sense to me but I leave that up to @inamoto4. The comment should be in `socialRelation`

File: data/Person.csv [hunk 2]
```diff
@@ -292,20 +292,22 @@ P0163,Li,Shu,en,male,NULL,NULL,NULL,NULL,1883,1954,,,PL0081,,,https://zh.wikiped
...
+P0168,小,霞,zh,,,,,,,,,,,,,SS00133,ONLINE SOURCE,,,,friend of P0052,2018-02-19,MG,2018-02-19,MG
+P0169,Wang,Hanying,en,,,,,,,,,,,,,SS00207,ONLINE SOURCE,,,,teacher of Mingxin 铭心 at Beijing No. 4 High School,2017-10-03,MG,2018-02-19,MG
+P0169,汪,含英,zh,,,,,,,,,,,,,SS00207,ONLINE SOURCE,,,,teacher of Mingxin 铭心 at Beijing No. 4 High School,2018-02-19,MG,2018-02-19,MG
+P0171,Teacher Ling,,en,,,,,,,,,,,,,SS00006,136?,,,,worked in the reference department of Jiangsu Provincial Song and Dance Troupe.,2017-10-03,MG,2018-02-19,MG
+P0171,凌老师,,zh,,,,,,,,,,,,,SS00006,136?,,,,worked in the reference department of Jiangsu Provincial Song and Dance Troupe.,2017-10-03,MG,2018-02-19,MG
 P0172,,Zhenzhen,en,,,,,,,,,,,,,,,,,,,2017-10-03,MG,2017-10-03,MG
-P0173,Liu,Nina,en,,,,,,,,,,,,,,,,,,,2017-10-03,MG,2017-10-03,MG
-P0173,刘,妮娜,zh,,,,,,,,,,,,,,,,,,,2017-10-03,MG,2017-10-03,MG
+P0173,Liu,Nina,en,,,,,,,,,,,,,SS00006,137,,,Professor at Department of Foreign Languages,,2017-10-03,MG,2018-02-21,MG
+P0173,刘,妮娜,zh,,,,,,,,,,,,,SS00006,137,,,Professor at Department of Foreign Languages,,2017-10-03,MG,2018-02-21,MG
```
Comment (duncdrum) [inline data/Person.csv]: `Membership`

File: data/Person.csv [hunk 3]
```diff
@@ -292,20 +292,22 @@ P0163,Li,Shu,en,male,NULL,NULL,NULL,NULL,1883,1954,,,PL0081,,,https://zh.wikiped
...
 P0174,Mao,Zedong,en,male,NULL,NULL,NULL,NULL,1893,1976,NULL,NULL,PL0080,NULL,NULL,https://en.wikipedia.org/wiki/Mao_Zedong,ONLINE SOURCE,,,,,2017-10-03,MG,2017-12-23,MG
 P0174,毛,泽东,zh,male,NULL,NULL,NULL,NULL,1893,1976,NULL,NULL,PL0080,NULL,NULL,https://en.wikipedia.org/wiki/Mao_Zedong,ONLINE SOURCE,,,,,2017-10-03,MG,2017-12-23,MG
-P0175,Peng,Yilin,en,,,,,,,,,,,,,,,,,,,2017-10-03,MG,2017-10-03,MG
+P0175,Peng,Yilin,en,,,,,,,,,,,,,SS00154,532,,,friend and middle school classmate of P0133,,2017-10-03,MG,2018-02-21,MG
```
Comment (duncdrum) [inline data/Person.csv]: `SocialRelation`

**Comment by inamoto4** [comment]
Very good, thank you. For your next branch please leave some space for my IDs.

**Pull request merged**